gr-azure-software-radio/README.md

```diff
@@ -52,7 +55,7 @@ resolve the problem.
 
 ```bash
 sudo apt-get install python3-pip cmake liborc-dev doxygen
-sudo pip install pytest pybind11
+sudo pip install pytest pybind11 azure-storage-blob azure-keyvault-secrets azure-identity azure-eventhub fastapi uvicorn
```

Comment:
these are in the requirements.txt file, do we need to specify them here? we can just point the user to run pip install -r python/requirements.txt